examples/centos/centos8-local.pkr.hcl

```diff
@@ -0,0 +1,82 @@
+packer {
+  required_plugins {
+   xenserver= {
+      version = "= v0.3.0-dev9"
```

Comment:
Please update all the examples to use a "real" release (v0.3.2). The `-dev` versions were for testing the new Github release process and shouldn't be used by an end user.

Reply:
Done